### ISSUE TITLE
`TextTruncator` component does not provide a way to read the hidden text

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#7557]
### Observed behavior

I noticed this on the Channel cards and breadcrumbs, but it's probably a pretty widespread issue.

Basically, if a long piece of text is truncated, I don't think there's a way for me to _read_ the full-text without navigating somewhere else.

Screenshots of places in the app. Sighted users cannot read the text hidden by "..." without navigating elsewhere. 

(The first one's not a good example because we're _in_ the topic and you see the header below, but imagine if you go a level deeper, then the breadcrumb would be truncated)

![image](https://user-images.githubusercontent.com/10248067/95923056-7f18f000-0d69-11eb-94b4-6f24e052e548.png)

![image](https://user-images.githubusercontent.com/10248067/95923108-9fe14580-0d69-11eb-9f5b-04dcdfbe2ce7.png)

![image](https://user-images.githubusercontent.com/10248067/95923111-a1ab0900-0d69-11eb-9743-ad8bae5e43ad.png)

### Expected behavior

I'm not sure how to handle this universally, but a general pattern is to give link-like things a `title` attribute so the full-text will be shown on most browsers when you hover over the link long enough.

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

**Comment by Kalovelo** [comment]
Hello once again @jonboiser, could a tooltip showing the remaining text like "...the rest of the details" do the trick here?It would kind of work like the link hovering. Other solutions like "view more" buttons would probably break the UI on components like cards/breadcrumbs...

**Comment by jonboiser** [comment]
I think one simple thing you can try is to put the full text inside the `title` attribute. This will make the text readable by just hovering the mouse over it for a few seconds:

Here's an example

https://codepen.io/jonboiser/pen/MWJwgEP?editors=1111

**Comment by Kalovelo** [comment]
Sweet! I will work on it and submit a PR if it is still up for grabs!

**Comment by pcenov** [comment]
Tested and verified as fixed as part of https://github.com/learningequality/kolibri/releases/tag/v0.14.7-rc1 testing.
